src/app/tests/suites/certification/Test_TC_DRLK_2_2.yaml

```diff
@@ -156,8 +156,6 @@ tests:
       PICS: DRLK.S.C00.Rsp
       command: "LockDoor"
       timedInteractionTimeoutMs: 1000
-      response:
-          error: 0x00
```

Comment:
Hmm...  Why is this change and similar below needed?

Reply:
This is mentions in the PR comment:

> Remove YAML test response checks for error == 0 (which is success). This was discuss with Vivien that YAML tests should not be written to check for error of 0, instead it should be completely removed for the implicit success check

I am also okay with going in the direction where the parse interprets `error == 0` as a success, but I find it odd for tests to say `error == 0` when implicitly the omission of the response entirely has always been to check for a success

Reply:
Ah, I see.  This makes sense, thanks.